web/src/lightbox.ts

```diff
@@ -44,7 +57,7 @@ export class PanZoomControl {
             $("#lightbox_overlay .lightbox-zoom-reset").removeClass("disabled");
         });
 
-        this.panzoom.on("panend", (e) => {
+        this.panzoom.on("panend", (e: PanZoom) => {
```

Comment:
Should `e` be an event, not a PanZoom object?

Reply:
Panzoom here is the instance which is passed as the parameter of the callback function not the Event. Here's the [reference](https://www.npmjs.com/package/panzoom#events-notification)